src/compiler/diagnosticMessages.json

```diff
@@ -1687,6 +1691,10 @@
         "category": "Error",
         "code": 2528
     },
+    "Duplicate identifier '{0}'. Compiler reserves name '{1}' in top level scope of a module containing async functions.": {

```

Comment:
This is modeled after TS2441: `Duplicate identifier '{0}'. Compiler reserves name '{1}' in top level scope of a module.` See: https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json#L1350